sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/aio/_form_training_client_async.py

```diff
@@ -146,16 +167,15 @@ async def train_model(
             **kwargs
         )
 
-        def callback(raw_response):
-            model = self._client._deserialize(Model, raw_response)
-            return CustomFormModel._from_generated(model)
-
-        deserialization_callback = cls if cls else callback
-        return await async_poller(
+        return AsyncLROPoller(
             self._client._client,
             response,
             deserialization_callback,
-            AsyncLROBasePolling(timeout=polling_interval, lro_algorithms=[TrainingPolling()], **kwargs)
+            AsyncLROBasePolling(  # type: ignore
```

Comment:
Why is there a type ignore comment here?

Reply:
error: Argument 4 to "AsyncLROPoller" has incompatible type "AsyncLROBasePolling"; expected "AsyncPollingMethod[Any]"